HPXMLtoOpenStudio/resources/hvac_sizing.rb

```diff
@@ -1570,8 +1572,7 @@ def self.apply_hvac_equipment_adjustments(hvac_sizing_values, weather, hvac, cfa
       end
       hvac_sizing_values.Heat_Airflow = calc_airflow_rate(hvac_sizing_values.Heat_Capacity, (hvac.SupplyAirTemp - @heat_setpoint))
 
-    elsif hvac.HeatType == HPXML::HVACTypeHeatPumpMiniSplit
-
+    elsif [HPXML::HVACTypeHeatPumpMiniSplit, HPXML::HVACTypeHeatPumpPTHP].include? hvac.HeatType
```

Comment:
I'm a bit surprised to see PTHP lumped in this MSHPs instead of ASHPs. Can you explain why that is? (I don't remember offhand why MSHPs have their airflow sized differently than ASHPs.)

Reply:
Yeah I am not certain about the reason why either, I guess at the beginning I am just confused by the different sizing equations, and then I found using ASHP equations need to set [`hvac.SupplyAirTemp`](https://github.com/NREL/OpenStudio-HPXML/blob/ptac-pthp-2/HPXMLtoOpenStudio/resources/hvac_sizing.rb#L1104-L1111). So I just chose the simpler one assuming they're similar. But I just ran a test comparing the sized airflow between using ashp equation and using mshp equation, I found they're not the same in the `hvac_sizing_values.Heat_Airflow` calculation in the following lines, one is 446.86(MSHP) while the other is 483.26(ASHP, setting 105F supply air temperature). I guess we will want to proceed with ASHP equations?